hazelcast/src/main/java/com/hazelcast/internal/serialization/SerializationService.java

```diff
@@ -42,6 +42,8 @@
 
     <T> T toObject(Object data);
 
+    <T> T bytesToObject(byte[] bytes, int dataOffset);
```

Comment:
What would be the purpose of the length? The deserialization step determines itself how much data it is going to read.